### PR TITLE
Added tooltips to all iframes

### LINKED ISSUE #369: toolTipText addition request
I think we need several tool tip text for the Preview, Documentation, Code window's buttons on title bar of respective windows, obviously the symbols are self-explainatory but a little addition can make it even convenient, which will be easy for the ones landing on the awesome WoofJS for the first time.
Let me know if we should incorporate this so I start working on it @stevekrouse .

### PR DESCRIPTION
**Pull request by Anshuman-Verma**
Fixes #369 
I have added tooltips to the buttons on respective iframes.
Kindly review @stevekrouse if they are up to the mark.

### CODE REVIEW

**Comment by stevekrouse** [comment]
**[This is an automated integration to preview this pull request's changes to the website.]**

https://cdn.rawgit.com/Anshuman-Verma/WoofJS/ee488c6d1950d6e45b7dd5b0226a8d5ba79c9879/index.html

File: create.html [hunk 1]
```diff
@@ -254,9 +254,9 @@
         <div id="PreviewNav">
           <span style="margin-left:8px; margin-top:4px">Preview</span>
           <span style="margin-right:8px">
-            <img onclick="runCode(); focusPreview()" id="pop" style="margin: 2px; width: 17px; height:17px" src="./images/refresh.png">
-            <img  onclick="popout()" id="pop" style="margin: 2px; width: 17px; height:17px" src="./images/pop.png">
-            <img  @click="previewbar = !previewbar"  style="margin: 5px; width: 16px; height:18px; cursor:pointer" src="./images/minimize.png">
+            <img onclick="runCode(); focusPreview()" id="pop" style="margin: 2px; width: 17px; height:17px" src="./images/refresh.png" title="Reload">
+            <img  onclick="popout()" id="pop" style="margin: 2px; width: 17px; height:17px" src="./images/pop.png" title="Maximize">
```
Comment (stevekrouse) [inline create.html]: Maybe change this one to "Open in new tab" or "Popout" or "New page". What do you think?

**Comment by stevekrouse** [comment]
Cool, I like how this looks!

File: create.html (hunk 1 above)
Comment (Anshuman-Verma) [inline create.html]: Alright, on it !

**Comment by Anshuman-Verma** [comment]
Be on for a while @stevekrouse while i fix the changes u requested, i kinda had the same doubt to make it 'open in new window' anyways i'll make the changes requested ASAP.

**Pull request merged**